src/modules/displaymanager/main.py

```diff
@@ -403,6 +401,11 @@ def run():
     # setup lightdm
     if "lightdm" in displaymanagers:
         if have_dm("lightdm", root_mount_point):
+            global lightdm_conf_path
```

Comment:
It was needed, since the value was reused at a second point. However I've fixed it now by readding the original code: https://github.com/calamares/calamares/pull/758/commits/e53d4b84dc138f02e6bca2f2591c1b862904d423